static/index.html

```diff
@@ -1,119 +1,108 @@
-                    <path d="M382 802 168 588l38-38 176 176 374-374 38 38-412 412Z" /></svg
-                ></button>
-                <!-- <aside></aside> -->
-              </div>
-            </td>
-          </tr>
+<head>
+  <title>DevPool Claims | Ubiquity DAO</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link href="out/styles/app.css" rel="stylesheet" />
+  <link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png" />
+  <link rel="icon" type="image/png" sizes="32x32" href="media/favicon.png" />
+</head>
 
-          <tr id="Allowance">
-            <th>Allowance</th>
-            <td id="permit.permitted.amount"></td>
-          </tr>
-          <tr id="From">
-            <th>From</th>
-            <td id="owner"></td>
-          </tr>
-          <tr id="Nonce">
-            <th>Nonce</th>
-            <td id="permit.nonce"></td>
-          </tr>
-          <tr id="Deadline">
-            <th>Deadline</th>
-            <td id="permit.deadline"></td>
-          </tr>
-          <tr id="Signature">
-            <th>Signature</th>
-            <td id="signature"></td>
-          </tr>
-        </tbody>
-      </table>
-    </main>
-    <footer>
-      <code></code>
-      <div id="build">
-        <a target="_blank" rel="noopener noreferrer"></a>
-      </div>
-    </footer>
-    <script></script>
+<body>
+  <background>
+    <div id="pre-grid"></div>
+    <div id="grid"></div>
+    <header>
+      <a href="http://ubq.fi/">
+        <div id="logo">
+          <div id="logo-icon">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 132 151" fill="currentColor">
+              <path
+                d="M132 41.1c0-2.3-1.3-4.5-3.3-5.7L69.4 1.2c-1-.6-2.1-.9-3.3-.9-1.1 0-2.3.3-3.3.9L3.6 35.4c-2 1.2-3.3 3.3-3.3 5.7v68.5c0 2.3 1.3 4.5 3.3 5.7l59.3 34.2c2 1.2 4.5 1.2 6.5 0l59.3-34.2c2-1.2 3.3-3.3 3.3-5.7V41.1zm-11.9 62.5c0 2.7-1.4 5.2-3.7 6.5l-46.6 27.5c-1.1.7-2.4 1-3.7 1s-2.5-.3-3.7-1l-46.6-27.5c-2.3-1.3-3.7-3.8-3.7-6.5V54.1c0-1.2.6-2.4 1.7-3 1.1-.6 2.3-.6 3.4 0l8 4.7c1.9 1.1 3 3.3 4.4 5.8.3.5.5 1 .8 1.4 3.5 6.3 5.2 13 6.8 19.5 3 11.9 6 24.2 21.3 28.2 5 1.3 10.4 1.3 15.4 0 15.2-4 18.3-16.3 21.3-28.2C96.8 76 98.5 69.3 102 63c.3-.5.5-1 .8-1.4 1.3-2.5 2.5-4.6 4.4-5.8l8-4.7c1-.6 2.3-.6 3.4 0s1.7 1.7 1.7 3v49.5zM62.6 13.7c2.2-1.3 4.9-1.3 7.1 0L110 37.6c1 .6 1.6 1 1.6 2.2 0 1.2-.6 1.9-1.6 2.5l-7.7 4.6c-3.4 2-5.1 5.2-6.6 8.1l-.1.2c-.2.4-.4.7-.6 1.1-3.8 6.8-6.6 14-8.2 20.4C83.6 89.1 82.4 97.3 72 100c-1.9.5-3.9.7-5.8.7-2 0-3.9-.3-5.8-.7C50 97.3 48.7 89.1 45.6 76.6 44 70.2 41.2 63 37.4 56.2c-.2-.3-.4-.7-.6-1l-.1-.3c-1.5-2.8-3.3-6.1-6.6-8.1l-7.7-4.6c-1-.6-1.6-1.3-1.6-2.5s.6-1.6 1.6-2.2l40.2-23.8z">
+              </path>
+            </svg>
+          </div>
+          <div id="logo-text"><span>Ubiquity DAO</span><span>DevPool Claims</span></div>
+        </div>
+      </a>
```

Comment:
Be sure to format with respect to space between brackets. In case you are not aware, if you have space between brackets, the DOM will render line breaks which causes precise designs to break apart.

I don't have time to setup and run this right now to verify. Its easier if you just make sure that any brackets that were touching before (like there) should be touching again in your submitted commit.